bessctl/conf/samples/chain4_profiler.bess

```diff
@@ -0,0 +1,138 @@
+#
+# Test NF chain #5: a stateful NF chain used by Dyssect
```

Comment:
Update a short comment section to say:
1) what the script does;
2) how it works;